src/assets/scss/components/_typography.scss

```diff
@@ -105,8 +105,8 @@ article {
     }
 }
 
-// Accented link with arrow
-a.link--accent {
+// Accented link with arrow right
+a.link--accent-right {
```

Comment:
You can extract common css properties into their own `link-arrow` class, and only append the extra properties you need in the `--left` and `--right` modifier, so that there's less to override.

In the markup this would look like `class="link-arrow link-arrow--left"`.